src/focus-visible.js

```diff
@@ -295,9 +295,7 @@ if (typeof window !== 'undefined' && typeof document !== 'undefined') {
   }
 
   window.dispatchEvent(event);
-}
 
-if (typeof document !== 'undefined') {
```

Comment:
I'm assuming these conditional blocks can be safely joined, for simplicity+brevity